requirements/base.txt

```diff
@@ -8,9 +8,17 @@ django-secure==1.0.1
 # Time zones support
 pytz==2015.4
 
+# mongo
+mongoengine==0.10.0
+
 # API
 djangorestframework==3.2.2
 django-rest-swagger==0.3.4
 drf-nested-routers==0.9.0
 
-requests==2.7.0
\ No newline at end of file
+requests[security]==2.7.0
```

Comment:
That's interesting. I've check my Python version and it's 2.7.10 and I still get the warnings if I don't install the security package. I'll investigate it more.